### PR TITLE
DS-12413: Correcting product name attribute.

### PR DESCRIPTION
**Pull request by grainnejenningsRH**
In the following sentence, 'OpenShift Data Science' is incorrect, and should just be OpenShift. 

>  If you decide to use specialized user groups without adding these groups from an identity provider, you must create the groups in OpenShift Data Science and then add the appropriate users to them. 

Checked with @chtyler, and replaced {productname-short}  with {openshift-platform}.

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: modules/overview-of-user-types-and-permissions.adoc [hunk 1]
```diff
@@ -51,7 +51,7 @@ endif::[]
 
 Optionally, if you want to restrict access to your {productname-short} deployment, you can create specialized user groups for users and administrators.
 
-If you decide to restrict access, and you already have user groups defined in your configured identity provider, you can add these user groups to your {productname-short} deployment. If you decide to use specialized user groups without adding these groups from an identity provider, you must create the groups in {productname-short} and then add the appropriate users to them.
+If you decide to restrict access, and you already have user groups defined in your configured identity provider, you can add these user groups to your {productname-short} deployment. If you decide to use specialized user groups without adding these groups from an identity provider, you must create the groups in {openshift-platform} and then add the appropriate users to them.
```
Comment (chtyler) [inline modules/overview-of-user-types-and-permissions.adoc:54]: Could shorten this to just:

If you decide to use specialized user groups without adding these groups from an identity provider, you must create the groups in {openshift-platform} and then add users to them.

File: modules/overview-of-user-types-and-permissions.adoc
Comment (chtyler) [inline modules/overview-of-user-types-and-permissions.adoc:1]: Your actual change looks good to me.

I can't seem to comment in-line for the parts of the file that you haven't changed. I'll write a few thoughts here:

1. We want to avoid using RHODS references in the code in the ODH repo. Ideally, remove the occurrences of "//For RHODS-1954". 
2. Occurrences of "if you are using" could be shortened to "if you use".
3. Occurrences of "are authenticated" could be changed to "authenticate", avoiding the passive voice. 
4. There are occurrences of "OpenShift" that could also use the {openshift-platform} variable. 
5. The following important note could reworded:

[IMPORTANT]
Although users of {productname-short} and its components are authenticated through OpenShift, session management is separate from authentication. This means that logging out of {openshift-platform} or {productname-short} does not affect a logged in Jupyter session running on those platforms. This means that when a user's permissions change, that user must log out of all current sessions in order for the changes to take effect.

The use of "This means" twice looks a little odd.  Also the words "..that user..." could be rewritten.

**Review by chtyler — CHANGES_REQUESTED** [review]
Your actual change looks good to me. I noticed a few other minor things that could be resolved.

**Comment by chtyler** [comment]
Also, the term "through OpenShift" could be changed to "on OpenShift"

File: modules/overview-of-user-types-and-permissions.adoc
Comment (grainnejenningsRH) [reply modules/overview-of-user-types-and-permissions.adoc:1]: Agreed that these would be more effective as broader changes in separate tickets.
